clients/src/main/java/org/apache/kafka/common/network/SslChannelBuilder.java

```diff
@@ -60,7 +58,6 @@ public SslChannelBuilder(Mode mode,
         this.mode = mode;
         this.listenerName = listenerName;
         this.isInterBrokerListener = isInterBrokerListener;
-        this.log = logContext.logger(getClass());
```

Comment:
maybe we should pass it to `Utils.closeQuietly` instead of deleting it.

Reply:
Since this field is unused so I thought we should just delete it.